File: app/src/main/java/com/qg/musicmaven/dreampage/DreamContract.java
package com.qg.musicmaven.dreampage;

import com.qg.musicmaven.BasePresenter;
import com.qg.musicmaven.BaseView;

/**
 * Created by jimji on 17-11-23.
 */

public interface DreamContract {

    interface View extends BaseView<Presenter> {

    }

    interface Presenter extends BasePresenter<View> {

    }

}

File: app/src/main/java/com/qg/musicmaven/ui/dreampage/DreamContract.kt
package com.qg.musicmaven.ui.dreampage

import com.qg.musicmaven.base.BaseView
import com.qg.musicmaven.modle.bean.Wish

/**
 * Created by jimiji on 2017/12/14.
 */
interface DreamContract {
    interface View : BaseView {
        fun loadMoreDone(list: MutableList<Wish>)
        fun reFreshDone(list: MutableList<Wish>)

    }
}